msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,26 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+To remotely debug your app running in Office app for Android.
```

Comment:
Whoops, this isn't a complete sentence. It's also a good place to incorporate the suggestion to mention Toolkit debugging isn't yet supported on mobile. How about this:

Teams Toolkit (`F5`) debugging is not yet supported with Office app for Android. Here's how to remotely debug your app running in Office app for Anrdroid: